csharp-advanced-topics/GenericsInCSharp/GenericsInCSharp.sln

```diff
@@ -0,0 +1,31 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
+VisualStudioVersion = 17.2.32630.192
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "GenericsInCSharp", "ConsoleApp1\GenericsInCSharp.csproj", "{57B9FF43-1FAA-4014-A90A-D385EA63FD27}"
```

Comment:
Can we change the folder name to `GenericsInCSharp` instead of `ConsoleApp1` please?